packages/framework-core/src/services/event-store.ts

```diff
@@ -50,6 +50,11 @@ export class EventStore {
     }
   }
 
+  public async destroyEntity(entityName: string, entityID: UUID): Promise<void> {
```

Comment:
Is this `async` needed? I don't see an await here `return this.provider.events.destroy(this.config, this.logger, entityName, entityID)`
```suggestion
  public destroyEntity(entityName: string, entityID: UUID): Promise<void> {
```

Reply:
Yeah is not needed!